tests/acceptance/cursor-movement-test.js

```diff
@@ -169,57 +170,62 @@ module('Acceptance: Cursor Movement w/ shift', {
   }
 });
 
-test('left arrow when at the end of a card moves the cursor across the card', assert => {
-  let mobiledoc = Helpers.mobiledoc.build(({post, cardSection}) => {
-    return post([
-      cardSection('my-card')
-    ]);
+if (supportsSelectionExtend()) {
```

Comment:
For some tests in the past we had a function `skipInPhantom` that ran the tests if they weren't running in phantom.
If there end up being multiple tests that shouldn't run for IE a `skipInIE` function or method on test: `test.skipIf(conditionFn, name, testFn)` would be useful to minimize the noise in diffs.